nft_ingester/src/processors/account_based/token_updates_processor.rs

```diff
@@ -36,6 +36,20 @@ impl TokenAccountsProcessor {
         res
     }
 
+    pub fn transform_and_save_fungible_token_account(
+        &self,
+        storage: &mut BatchSaveStorage,
+        key: Pubkey,
+        token_account: &TokenAccount,
+    ) -> Result<(), StorageError> {
+        self.save_token_account_with_idxs(storage, key, token_account)?;
+
+        storage.fungible_asset_updated_with_batch(
+            token_account.slot_updated as u64,
+            token_account.pubkey,
+        )
+    }
+
```

Comment:
Fixed

Reply:
Do you mean `transform_and_save_token_account` should call `fungible_asset_updated_with_batch`? Afaik it shouldn't, because it's a method for non-fungibles only. @RequescoS send help pls

Reply:
There are a beezillion fungibles on Solana and other chains. We're calling the new method only for 2 major fungibles on Solana

Reply:
Yeah, I get it. I told @RequescoS that we should've left that invocation here, however, he said we should not. @RequescoS please correct me if I'm wrong.

Reply:
I believe we've updated that line during the paired review, haven't we?

Reply:
somehow GitHub was not rendering the changes properly, or I've used some weird cached version, all is good